apps/crn-server/src/routes/research-outputs.route.ts

```diff
@@ -121,20 +122,30 @@ export const researchOutputRouteFactory = (
   researchOutputRoutes.put(
     '/research-outputs/:researchOutputId',
     async (req, res) => {
-      const { body, params, loggedInUser } = req;
+      const { body, params, loggedInUser, query } = req;
       const { researchOutputId } = validateResearchOutputParameters(params);
       const updateRequest = validateResearchOutputPutRequestParameters(body);
       validateResearchOutputPostRequestParametersIdentifiers(body);
+      const options = validateResearchOutputRequestQueryParameters(query);
+      const publish = options.publish ?? true;
```

Comment:
Should the default be false so that we don't publish be mistake?

Reply:
I based this off the post request which is publish true by default, I get your point but then I'd say we should probably change it there as well and in the controllers and data providers as well. I think that would warrant another ticket.